packages/appstore/src/components/routes/routes.tsx

```diff
@@ -24,18 +21,9 @@ const Routes: React.FC = observer(() => {
             <Switch>
                 {getRoutesConfig({
                     consumer_routes: config.routes,
-                }).map((route: TRoute, idx: number) => {
-                    // Temporary way to intercept the route to show the Wallets component.
```

Comment:
Revert back the feature flag changes in `@deriv/appstore` routing since we don't serve `@deriv/wallets` here anymore.